README.md

```diff
@@ -76,13 +355,22 @@ config =
 Sparrow.H2Worker.Pool.start_link(:your_apns_workers_name, config)
 ```
 
-## How to obtain device token??
+### How to obtain a device token?
 
 Try [this](https://developer.apple.com/documentation/usernotifications/registering_your_app_with_apns).
 
-## How to obtain apns-topic??
+### How to obtain an apns-topic??
 
 Last but not least, to get the 'apns-topic' header value, go to:
 Xcode -> open your swift app -> General -> Identity -> Bundle Identifier
 
 Good luck :)
+
+# TODO
```

Comment:
Is the readme the right place to put non-feature todos?

Reply:
I'll remove this section